From 0136bbcb0e85581967aec07b55d5bb2d230fa0a6 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 19:57:12 +0100
Subject: [PATCH 1/8] ia64: Add statmount() and listmount() syscalls

statmount() and listmount() were added after ia64 was removed from
the kernel.

Fixes: d8b0f5465012 ("wire up syscalls for statmount/listmount")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 6db5e2f51435..9064b3c8128d 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -377,3 +377,5 @@
 454	common	futex_wake			sys_futex_wake
 455	common	futex_wait			sys_futex_wait
 456	common	futex_requeue			sys_futex_requeue
+457	common	statmount			sys_statmount
+458	common	listmount			sys_listmount
-- 
2.52.0

